.devcontainer/Dockerfile

```diff
@@ -18,6 +18,6 @@ RUN bash /tmp/library-scripts/common-debian.sh "${INSTALL_ZSH}" "${USERNAME}" "$
     && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/library-scripts
 
 # [Optional] Uncomment this section to install additional OS packages.
-# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-#     && apt-get -y install --no-install-recommends <your-package-list-here>
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends gettext-base
```

Comment:
Did you find that this was required?

Reply:
Yeah - didn't work for the fics stuff initially. Got "command not found". With this added and the container rebuilt it was fine.

Reply:
Oh, cool!  Thanks a lot for fixing this.

We have some az cli changes coming soon to help, but won't be full E2E story.  

I think the first rev will be replacing some of the calls, like to graph.